spec/factories/users.rb

```diff
@@ -13,8 +13,16 @@
 
     trait :mentor do
       mentor true
+      mentor_industry {["Business", "Technology", "Startup"].sample}
     end
 
+    trait :mentee do
+      wants_mentor true
```

Comment:
might be worthwhile to simply make this part of 'mentee' factory instead, see previous